src/Epochs/utc.jl

```diff
@@ -0,0 +1,90 @@
+const LEAP_J2000 = round.(Int, (LeapSeconds.LS_EPOCHS .- value(J2000_TO_MJD)) * 86400)
+const LEAP_TAI = LEAP_J2000 .+ round.(Int, LeapSeconds.LEAP_SECONDS) .- 1
+const LEAP_TAI_SET = Set(LEAP_TAI)
+
+function from_utc(str::AbstractString;
```

Comment:
This is the headline change. There is no `Epoch{UTC}` anymore but we can still convert to/from UTC via the dedicated `from_utc` and `to_utc`. The discussion about this is in https://github.com/JuliaAstro/AstroTime.jl/issues/50#issuecomment-662049786.

**TL;DR:** The `Epoch` data format is incompatible with a discontinuous timescale like UTC (and only UTC 👎 ). Thus, it makes no sense to rewrite the whole library and add a lot of edge cases for UTC. The best way of dealing with UTC and leap seconds is by keeping them on the outer edge of your code or pretend they do not exist like Google (https://github.com/google/unsmear). *Friends do not let friends use UTC for mission-critical time keeping.*